docs/basics/1d_hist.rst

```diff
@@ -218,3 +218,99 @@ To compare two histograms, using the histograms defined above:
    :alt: Simple hist
    :width: 500
 
+
+Comparing two histograms (advanced)
+===================================
+
+In this example, we will compare two tuple of histograms and use pull and diff comparisons.
+
+First, we make the histograms and scale them:
+
+.. code-block:: python
+
+    from plothist import (
+        make_hist,
+        plot_hist,
+        plot_error_hist,
+        plot_comparison,
+        create_comparison_figure,
+    )
+
+    name = "variable_1"
+    category = "category"
+
+    x1 = df[name][df[category] == 1]
+    x2 = df[name][df[category] == 4]
+    x3 = df[name][df[category] == 3]
+    x4 = df[name][df[category] == 5]
+
+    x_range = [-9, 9]
+
+    h1 = make_hist(x1, bins=50, range=x_range)
+    h2 = make_hist(x2, bins=50, range=x_range)
+    h3 = make_hist(x3, bins=50, range=x_range)
+    h4 = make_hist(x4, bins=50, range=x_range)
+
+    # Scaling
+    scaling = max([h1.sum().value, h2.sum().value, h3.sum().value, h4.sum().value])
+    h1 *= scaling / h1.sum().value
+    h2 *= scaling / h2.sum().value
+    h3 *= scaling / h3.sum().value
+    h4 *= scaling / h4.sum().value
+
+
+Then, we plot the histograms and the comparison plots on different axes:
+
+.. code-block:: python
+
+    # Create the 3 axes that we need for this plot
+    fig, axes = create_comparison_figure(
+        figsize=(6, 6), nrows=3, gridspec_kw={"height_ratios": [5, 1, 1]}
+    )
+
+    # Assign each axes: 1 to plot the histograms and 2 for the comparison plots
+    ax_main, ax1_comparison, ax2_comparison = axes
+
+    # Here, we use step as a histtype to only draw the line
+    plot_hist(h1, label="Train A", ax=ax_main, histtype="step", linewidth=1.2)
+    plot_hist(h3, label="Train B", ax=ax_main, histtype="step", linewidth=1.2)
+    # And then, to make the plot easier to read, we redraw them with stepfilled, which add color below the line
+    plot_hist(h1, ax=ax_main, histtype="stepfilled", color="#348ABD", alpha=0.2)
+    plot_hist(h3, ax=ax_main, histtype="stepfilled", color="#E24A33", alpha=0.2)
```

Comment:
Can we save lines with
https://stackoverflow.com/questions/28398200/plotting-transparent-histogram-with-non-transparent-edge

Reply:
I'm not a fan on this complicated RGB no-one ever used `fc` argument vs this easy to read easy to understand easy to replicate example.

Reply:
Simpler: `plot_hist(h1, ax=ax_main,  histtype="stepfilled", color="#348ABD30", edgecolor="#348ABD", linewidth=1.2)`